docs/.vuepress/config.js

```diff
@@ -1,55 +1,86 @@
 module.exports = {
-  title: "Shopware-PWA",
+  title: "PWA",
   description: "Headless PWA for Shopware",
   themeConfig: {
     repo: "DivanteLtd/shopware-pwa",
     docsDir: "docs",
-    sidebarDepth: 2,
+    sidebarDepth: 0,
     smoothScroll: true,
-    sidebar: {
-      "/cli/": [""],
-      "/api/": [
-        {
-          title: "API",
-          path: "",
-          collapsable: false,
-          children: [
-            { title: "Composables", path: "composables" },
-            { title: "Helpers", path: "helpers" },
-            { title: "Shopware 6 client", path: "shopware-6-client" },
-          ],
-        },
-      ],
-      "/": [""],
-    },
+    displayAllHeaders: true,
+    logo: "/shopware_docs_logo.svg",
+    sidebar: [
+      {
+        title: "Home",
+        path: "/",
+        collapsable: true,
+        children: [
+          ["/landing/project/", "Project"],
+          ["/landing/getting-started/", "Getting started"],
+          ["/landing/fundamentals/", "Fundamentals"],
+          ["/landing/concepts/", "Concepts"],
+          ["/landing/operation/", "Operation"],
```

Comment:
Consider using plural - `Operations`. Not urgent and I am not sure, which one looks better. :p

Reply:
```suggestion
          ["/landing/operation/", "Operations"],
```